### DISCUSSION TITLE
TEA Stake Validators

### DISCUSSION THREAD

**Discussion by newgamehd02** [teaxyz/white-paper#117] [Q&A] [answered]
Will there be more than one validator for staking? How will validator elections take place? Will only OSS developers have validation rights? Or will everyone who wants to have the right to validate, like Ethereum?

**Comment by Abdulra850** [comment]
I believe there will be multiple validators.

**Comment by SesiomRule** [comment]
i think if  they made  oss dev as a  validator , they only make it to handle the situation, we'll see the answer in the future coming from devs

**Comment by 0xsingular** [comment, marked as answer]
The tea protocol is built on Base (L2), it is not a separate blockchain, i.e. no validators are needed to build/verify blocks. Even though the tea protocol is based on the Proof of Contribution algorithm, you can still draw a parallel between validators and "tea testers" who review and validate packages, and here too there is a balance between reward and penalty.
Regarding staking, the staking mechanism is used within the tea protocol to decentralize the reputation system, so you choose which package to stake your TEA tokens in favor of and in this way contribute to the decentralization of this reputation and reward system.
For reference you can read [the whitepaper](https://info.tea.xyz/tea-white-paper/white-paper) or in a compact form [visit the tea website](https://tea.xyz/).

**Comment by hatafmuhammad0** [comment]
> Will there be more than one validator for staking? How will validator elections take place? Will only OSS developers have validation rights? Or will everyone who wants to have the right to validate, like Ethereum?

Multiple validator is a good option.
